discovery/targetgroup/targetgroup_test.go

```diff
@@ -59,6 +59,39 @@ func TestTargetGroupStrictJsonUnmarshal(t *testing.T) {
 	}
 }
 
+func TestTargetGroupJsonMarshal(t *testing.T) {
```

Comment:
nit: `Json` should be `JSON` in Go style, but I realize it's the same for the test above and below...

Reply:
Thanks, I changed both to be consistent.